examples/Python/PET/acquisition_data.py

```diff
@@ -170,6 +167,8 @@ def main():
     tmx = image.transf_matrix()
     print(tmx)
 
+    msg_red.quit()
```

Comment:
We didn't have this before. Is this needed? If so, we'd need it in the SIRF-Exercises as well.

Reply:
I added this dummy method solely to get rid of Codacy annoying complaints about `msg_red` not being used - cf. my comment on line 224 of .../pSTIR/STIR.py.

Reply:
hmmm. I find cluttering the examples to satisfy Codacy not optimal. I'm not sure how to disable this error (I suppose we could configure Codacy to remove all warnigns about unused variables, but that seems worse!). Possibly @casperdcl knows a way?

Otherwise, I'd just remove them, and live with the Codacy warning. sorry.

Reply:
you can do any one of:

- configure the linters which Codacy uses:
  ```diff
  -msg_red = MessageRedirector()
  +msg_red = MessageRedirector()  # pylint:disable=unused-variable
  ```
- fix the issue
  ```diff
  -msg_red = MessageRedirector()
  +_ = MessageRedirector()
  ```
- ignore false-positives

Reply:
@casperdcl `_=` works, thanks a lot!